### ISSUE TITLE
[Feature Request] Ignore specific user messages in groups if its blocked

### ISSUE THREAD

**Issue by GnubiBORED** [telegramdesktop/tdesktop#6452]
Title, if someone is blocked by you, all messages should be ignored in groups or be able to hide them specifically

**Comment by RememberTheAir** [comment]
Duplicate of #1281, #1025, #5492. Although Telegram's Twitter said they might introduce it

**Comment by 411an** [comment]
> Although Telegram's Twitter said they might introduce it

Could you show me the link?

I want to go there to try hurry them up.

Six, six damn years we are asking it!

**Comment by RememberTheAir** [comment]
@411an iirc I think I've linked the tweet in one of the issues mentioned in my previous comment
